content/en/examples/access/validating-admission-policy-audit-annotation.yaml

```diff
@@ -12,4 +12,5 @@ spec:
       resources:   ["deployments"]
   validations:
     - key: "high-replica-count"
-      valueExpression: "object.spec.replicas > 50 ? 'Deployment spec.replicas set to ' + string(object.spec.replicas) : null"
+      expression: "object.spec.replicas > 50"
+      messageExpression: "'Deployment spec.replicas set to ' + string(object.spec.replicas)"
```

Comment:
How do we know this is what was intended? (it looks plausible, but I'm not sure).

Reply:
I read the docs and checked the API specs.